oat-sa/extension-tao-proctoring
# fix: Japanese translations

## Ticket
https://oat-sa.atlassian.net/browse/CGF-427

## Summary
Japanese translation fixed

## How to test (guide shared across the related PRs)
1. Apply all 11 PRs from the ticket
2. taoInstall with the following extensions: 
* taoCe
* ltiProctoring
* taotestCenter
* taoAltResultStorage
* taoBlueprints
* taoClientRestrict
* taoColorContrast
* taoOutcomeRds
* taoResourceWorkflow
* taoTaskQueue
* taoThemingPlatform
* xmlEditRp

3. Go to the BO, publish a new delivery and check its property ‘Allow Proctoring’ is in English
4. You may find the other places to check in the ticket
5. Now revert all PRs, taoInstall with the same extension list, verify the bug actually reproduce
